### PR TITLE
Alina

### PR DESCRIPTION
**Pull request by alinaincodeland**

### CODE REVIEW

**Review by ju8ko — APPROVED** [review]

File: client/src/pages/TeacherLessons.jsx [hunk 1]
```diff
@@ -3,28 +3,43 @@ import LessonCard from "../components/LessonCard";
...
 const TeacherLessons = () => {
-  const [showCreateLessonModal, setShowCreateLessonModal] = useState(false);
+  const [optModal, setOptModal] = useState(false);
+  const toggleShow = () => setOptModal(!optModal);
 
-  const handleCreateLesson = () => {
-    setShowCreateLessonModal(true);
+  const handleCreateLessonConfirm = () => {
+    setOptModal(true);
+    // save changes
```
Comment (Rami24t) [inline client/src/pages/TeacherLessons.jsx:15]: plz add reset form

File: client/src/components/lessonCard.css [hunk 2]
```diff
@@ -58,4 +58,29 @@
...
+
+}
+
+.lesson-card .lesson-card-icon {
+    padding-top: 0px;
+    line-height: normal;
+    color: green; 
+}
+
+.form-label {
```
Comment (Rami24t) [inline client/src/components/lessonCard.css:84]: plz add .lesson-card before

**Review by Rami24t — CHANGES_REQUESTED** [review]
please make some changes

**Review by Rami24t — APPROVED** [review]
Gut gemacht!

**Pull request merged**